tiledb/common/algorithm/test/unit_parallel_merge.cc

```diff
@@ -1465,3 +1475,35 @@ TEST_CASE("parallel merge example", "[algorithm][parallel_merge]") {
     CHECK(output == expect);
   }
 }
+
+#if defined(TILEDB_INTERCEPTS)
+
+TEST_CASE("parallel merge destructor race", "[algorithm][parallel_merge]") {
```

Comment:
Is this test useful now that the INTERCEPT macros are not present in the parallel merge calls?